codersdk/workspacesdk/connector.go

```diff
@@ -75,6 +78,8 @@ func runTailnetAPIConnector(
 		connected:     make(chan error, 1),
 		closed:        make(chan struct{}),
 	}
+	// TODO: reenable in upstack pr
+	// conn.SetTunnelDestination(agentID)
```

Comment:
This should be set immediately before sending a tunnel request or `SubscribeAgent()`.